projects/components/header/src/header.directives.ts

```diff
@@ -0,0 +1,16 @@
+import { Directive } from '@angular/core';
+
+@Directive({
+  selector: '[psHeaderTopButtonSection]',
```

Comment:
maybe we should be more careful when naming template directives. in ps-table we use a mix of Template, Section and (nothing) as suffix. (*psTableColumnHeaderTemplate, *psTableTopButtonSection, *psTableListActions, etc.)

Reply:
We should talk about a convention for directive names and fix them all at once.